codespell_lib/data/dictionary.txt

```diff
@@ -6781,6 +6781,8 @@ reacing->reaching
 reacll->recall
 readabilty->readability
 readapted->re-adapted
+readded->read
```

Comment:
http://www.yourdictionary.com/readd

Reply:
hah, didn't read the word that way. Nice catch. How about:
```
readded->readded, read,
readed->read, readd, readded,
```

Reply:
sounds good

Reply:
ok.. revised an push -f 'ed